src/configure/configure.ts

```diff
@@ -356,7 +356,7 @@ class Orchestrator {
             remoteUrl: '',
             repositoryId: '',
             repositoryName: '',
-            repositoryProvider: RepositoryProvider.AzureRepos
+            repositoryProvider: RepositoryProvider.Github,
```

Comment:
This should be controlled via a VS Code [setting](https://code.visualstudio.com/docs/getstarted/settings). That setting will allow user to switch back to Azure repo if they want. This also provides us a fallback if customers start hitting issue with new flow of GitHub repo.

Reply:
See one such setting here https://github.com/microsoft/vscode-deploy-azure/blob/c757820f632a2c4fc02fccefe2204dd9f29470aa/src/configure/model/models.ts#L21

Reply:
Okay, will add this setting.